event.go

```diff
@@ -209,6 +209,15 @@ func (e *Event) Object(key string, obj LogObjectMarshaler) *Event {
 	return e
 }
 
+// Func allows an anonymous func to run only if the event is enabled.
+func (e *Event) Func(f func(e *Event)) *Event {
+	if e == nil {
+		return e
+	}
+	f(e)
```

Comment:
You still have to check if enabled as an event can be non nil with the level `Disabled`.

Reply:
Fixed.